Intermittent failures for CLI acceptance tests

This test sometimes passes and sometimes fails.

    go test github.com/mitchellh/packer/packer

When it fails, the output is inconsistent.

Example 1:

```
--- FAIL: TestRemoteCmd_StartWithUi-8 (0.00s)
	communicator_test.go:37: bad output: 'hello




		world


		there
		'
```

Example 2:

```
--- FAIL: TestRemoteCmd_StartWithUi-8 (0.00s)
	communicator_test.go:37: bad output: 'hello


		world
		there
		'
```

Example 3:

```
--- FAIL: TestRemoteCmd_StartWithUi-8 (0.00s)
	communicator_test.go:37: bad output: 'hello
		world


		there
		'
```

Occasionally it works fine:

```
ok  	github.com/mitchellh/packer/packer	0.226s
```

--- a/packer/communicator.go
+++ b/packer/communicator.go
@@ -1,11 +1,12 @@
 package packer
 
 import (
-	"github.com/mitchellh/iochan"
 	"io"
 	"os"
 	"strings"
 	"sync"
+
+	"github.com/mitchellh/iochan"
 )
 
 // RemoteCmd represents a remote command being prepared or run.
@@ -132,9 +133,13 @@ OutputLoop:
 	for {
 		select {
 		case output := <-stderrCh:
-			ui.Message(r.cleanOutputLine(output))
+			if output != "" {
+				ui.Message(r.cleanOutputLine(output))
+			}
 		case output := <-stdoutCh:
-			ui.Message(r.cleanOutputLine(output))
+			if output != "" {
+				ui.Message(r.cleanOutputLine(output))
+			}
 		case <-exitCh:
 			break OutputLoop
 		}

